travis-ci/managers/fuzzit.sh

```diff
@@ -33,7 +33,7 @@ fi
 # Because we want Fuzzit to run on every pull-request and Travis/Azure doesnt support encrypted keys
 # on pull-request we use a write-only key which is ok for now. maybe there will be a better solution in the future
 FUZZIT_API_KEY=7c1bd82fe0927ffe1b4bf1e2e86cc812b28dfe08a7080a7bf498e98715884a163402ee37ba95d4b1637247deffcea43e
-FUZZIT_ADDITIONAL_FILES="./out/src/shared/libsystemd-shared-242.so"
+FUZZIT_ADDITIONAL_FILES="./out/src/shared/libsystemd-shared-243.so"
```

Comment:
@yuwata thank you! I think it would be better to let bash expand that part for us to prevent this from happening again. Could you replace "243" with "*"?

Reply:
OK. Done.